.github/workflows/deploy.nightly.devnet.yml

```
@@ -125,24 +141,39 @@ jobs:
                           --premine 0xA39Fed214820cF843E2Bcd6cA1759257a530894B:1000000000000000000000000000 \
                           --premine 0x181d9fEc79EC674DD3cB30dd9dd4188E737939FE:1000000000000000000000000000 \
                           --premine 0x1AB8C3df809b85012a009c0264eb92dB04eD6EFa:1000000000000000000000000000 \
-              --block-gas-limit {{ block_gas_limit }} --block-time {{ block_time }}s \
+                          --block-gas-limit {{ block_gas_limit }} --block-time {{ block_time }}s \
+                          --block-gas-limit {{ block_gas_limit }} --block-time {{ block_time }}s \
```

Is one of those redundant?
```suggestion
                          --block-gas-limit {{ block_gas_limit }} --block-time {{ block_time }}s \
```

---

oops this is deleted